avocado/utils/vmimage.py

```diff
@@ -280,7 +280,7 @@ class DebianImageProvider(ImageProviderBase):
 
     name = 'Debian'
 
-    def __init__(self, version='[0-9]+.[0-9]+.[0-9]+-.*', build=None,
+    def __init__(self, version='[0-9]+.[0-9]+.[0-9]+.*', build=None,
```

Comment:
Is this a version or a regex? IIUC, could this be `version_regex` instead of `version`?

Reply:
It's not always a regex AFAICT... and that would break change too much of the API at this point, IMO, to be justifiable.

Reply:
ok, no problem.